tests/Extensions/Diagnostics.HealthChecks.UnitTests/HealthCheckPublisherTests.cs

```diff
@@ -474,7 +475,8 @@ public override void Configure(bool closed = false)
 
 		internal class RestPublisherTestContext : PublisherTestContext
 		{
-			public Mock<IServiceFabricClient>? SFClient { get; protected set; } // TODO: move to RestTestContext
+			public Mock<IServiceFabricClient>? SFClient { get; protected set; }
+			public Mock<IServiceFabricClientWrapper>? SFClientWrapper { get; protected set; }
```

Comment:
nit: I don't see any point in making this property public or even having those properties for now, especially since Publisher field internal for some reason.